cilium: add visibility for all flags in CT dump

borkmann opened the pull request:
While debugging #10942, I noticed that not all flags are currently
dumped in our bugtool report's CT list. Fix it by adding the missing
ones as well.

Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>

borkmann committed:
cilium: add visibility for all flags in CT dump

While debugging #10942, I noticed that not all flags are currently
dumped in our bugtool report's CT list. Fix it by adding the missing
ones as well.

Also dump all unknown flags, so it is easier visible that we're
missing something.

Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>

tklauser commented on pkg/maps/ctmap/types.go:
```diff
@@ -487,12 +487,14 @@ type CtEntry struct {
+	SeenNonSyn    = 1 << 4
+	NodePort      = 1 << 5
+	ProxyRedirect = 1 << 6
+	DSR           = 1 << 7
```
Nit: could use `iota` to define this and make future additions easier:

```suggestion
	RxClosing     = 1 << iota
	TxClosing
	Nat64
	LBLoopback
	SeenNonSyn
	NodePort
	ProxyRedirect
	DSR
```

borkmann replied:
Ah yeah, good point, will change it into this style.

joestringer reviewed (commented):
For future proofing, we should probably consider matching on any currently-unknown flags and printing their hex value so that we actually know that we're missing the translation for them but we will actually have the value :-)

borkmann committed:
tests, services: show after how many requests we failed to connect

It would be useful to know whether we failed right away to connect to
a service or only after a certain number of attempts.

Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>

aanm reviewed (approved):
LGTM once Tobias comment is solved.
